### ISSUE TITLE
issues: open new issue on issue list

### ISSUE THREAD

**Issue by skanehira** [skanehira/gh.vim#85]
## 👔 Overview
Use `ghn` open creating issue buffer
Do this after #83

## 👖 How to achieve
Add key map `ghn`

## 🧦 Steps
1. Open `gh://:owner/:repo/issues`
2. Press `ghn`
3. Choise how to open buffer
4. Open `gh://:owner/:repo/issues/new`
